guide/admin/usergroups.md

```diff
@@ -0,0 +1,53 @@
+## User Handbook
+### Creating, Deleting and Modifying User Groups
+
+Managing user groups on your forum instance requires you to be in the Administrator Control Panel, you can do this by clicking your username to the top right of your screen, and then clicking **Administration**. The example below will show you how this is done:
+
+![Administration Link Screenshot](687474703a2f2f692e696d6775722e636f6d2f575a315a6a78392e676966.gif)
+
+Once you are in the Administration Panel, you'll see some navigation tabs flowing down the left hand side, click on **Permissions** to reveal the configuration page to the right hand side.
+
+![Permissions Screenshot](687474703a2f2f692e696d6775722e636f6d2f795834455466302e706e67.png)
+
+User groups are used for all kinds of different purposes, Flarum has a fantastic management system which makes it super-easy to manage user groups, and their associated permissions.
+
+### Create a new User Group
+
+On the configuration page, you'll see some pre-created user groups (if your Flarum installation is a fresh install), these are Admins & Mods, as shown below:
+
+![SS Group bar](687474703a2f2f692e696d6775722e636f6d2f30345a457730472e706e67.png)
+
+Next to those user groups, you'll see **New Group** click that to bring up the **Create Group** prompt.
+
+![SS Group Creation Prompt](687474703a2f2f692e696d6775722e636f6d2f4d4d3039314f4a2e706e67.png)
+
+There are 4 pieces of information Flarum needs from you to create this new user group, for this example we're going to be creating a user group called **Staff Member** but you can name this anything you wish.
+
+### Create Group Form Components
+
+#### Name
+
+ - Singular: is how the name is written should it only apply to one person, in this example our Singular name is **Staff Member**
+ - Plural: is how the name is written should it apply to more than one person, this example would be named **Staff Members**
+ 
+#### Color
+
+Choosing a Color can be done by obtaining the HTML Colour Code of your desired colour, there are plenty of websites you can view which a quick Google search would give you. Enter the colour code you wish to use (starts with a **#**).
```

Comment:
Ok that can happen, that is just the brit in me coming out, we spell that differently over here. I will change all mentions of it to be Color on edit.